### ISSUE TITLE
Feedback 'APK size'

### ISSUE THREAD

**Issue by xxfast** [xxfast/JetPackSavesThePlanet#95]
### Message from Stephen Muehlenberg sent on 2017-10-04 01:25:12 UTC###

> I noticed the Android APK is over 43 Mb, which is pretty big. Most games I see on the store are closer to 30, and yours is a pretty small game. See if you can minimise the size, possibly by removing unused assets or compressing assets where possible. Music and sounds are likely culprits.

### Link to HockeyApp ###

* [https://rink.hockeyapp.net/manage/apps/624899/feedback/1263243](https://rink.hockeyapp.net/manage/apps/624899/feedback/1263243)

**Comment by 6819710** [comment]
we can potenually save some space by downing the quality of some of the sound assets, but the file sizes of the assests are alredy max compressed and only take up arround 10 or so mb of the APK size (if unity is correct on sizes)

I do agree though a general cleanup would be good.

**Comment by 6819710** [comment]
https://docs.unrealengine.com/latest/INT/Platforms/Android/ReducingAPKSize/

**Comment by 6819710** [comment]
https://docs.unity3d.com/Manual/ReducingFilesize.html

**Comment by xxfast** [comment]
Yeah, I investigated this issue, and it seems that the sound assets are the culprit. We might want to compress the RAW soundtracks and the SFXs

**Comment by 6819710** [comment]
they audio assets already are compressed by unity. the ratio been like 10-14% of original size
